src/index.js

```diff
@@ -1,5 +1,9 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+import Router from "./Router";
+
+import "./style/common.scss";
+import "./style/reset.scss";
```

Comment:
```suggestion
import "./style/reset.scss";
import "./style/common.scss";
```

reset이 먼저 import 될 수 있도록 해주세요